### PR TITLE
Update snake and apple UI

### PR DESCRIPTION
**Pull request by Jordan-Zhu**
#1

Changed snake to draw the body and tail. Since the head was being drawn at the last set of coordinates in snakeList, I assumed [0, 0] would be the end of the list. So that's where I draw the snake's tail. 

Also, I wasn't able to test the code, school computer doesn't allow installing... I followed the convention from elsewhere in the code so if that stuff works, I'm guessing this shouldn't have a problem either. 

Apple and snake head by 🖌 @brucecantarim, I merely added them to assets, since they were so well done, and made modifications for the body and tail :p

### CODE REVIEW

**Comment by amarlearning** [comment]
Hey @Jordan-Zhu, LGTM :smile: 
You forgot to load these images in the game .. somewhere here https://github.com/Jordan-Zhu/Plaked/blob/master/mainPlaked.py#L39

Let me know if you need any help!

**Comment by codegateau** [comment]
When the snake dies . Try to put the snake blinking as indication that the snake had died.

https://github.com/amarlearning/Plaked/blob/master/mainPlaked.py#L272

**Comment by amarlearning** [comment]
can you please correct this! 
screenshot from 2016-10-31 01-44-17

File: mainPlaked.py [hunk 1]
```diff
@@ -147,10 +153,36 @@ def snake(block, snakeList):
...
+			gameDisplay.blit(body, (XnY[0], XnY[1]))
+			# pygame.draw.rect(gameDisplay, green, [XnY[0], XnY[1], block, block])
+		# Since the head seems to be at the last array element,
+		# Drawing tail at the start of the array
+		gameDisplay.blit(tail, (snakeList[0][0], snakeList[-1][1]))
+	else:
+		# This intends to delete the snake starting from the tail when the snake eats itself
+		for i in range(0, 3):
+			# Give it a flashing effect
+			for i % 2 == 0:
```
Comment (amarlearning) [inline mainPlaked.py]: I don't think it's a valid syntax

Comment (Jordan-Zhu) [reply mainPlaked.py]: Oops, meant to use 'if'. I'll go change that

**Comment by Jordan-Zhu** [comment]
Hmm some future ideas... 
- Having the snake's body follow its head by keeping a list of directions (only as long as the snake) and using those to turn the snake when it reaches the point where the head changed direction
- Grid based background for context while moving
- On game over state, pause movement to show the death animation

Edit: typos

**Comment by Jordan-Zhu** [comment]
I'm doing this for hacktoberfest so unfortunately -- being real -- I'm not able to work on it much after whenever the event ends. 

(One last thing though, fixing the snake's body so it's block x width and it shouldn't have those gaps)

Cheers! :tada:

**Comment by amarlearning** [comment]
@Jordan-Zhu thanks for your contribution!
I will work on that if get some time :smile: 

i'll merge the request ASAP

**Comment by codegateau** [comment]
This isnt copyrighted? i want to clone it

**Comment by amarlearning** [comment]
@codegateau go ahead! :smile:

**Pull request merged**